bgpd/bgp_clist.c

```diff
@@ -878,12 +946,16 @@ static int community_list_dup_check(struct community_list *list,
 
 /* Set community-list.  */
 int community_list_set(struct community_list_handler *ch, const char *name,
-		       const char *str, int direct, int style)
+		       const char *str, const char *seq, int direct, int style)
 {
 	struct community_entry *entry = NULL;
 	struct community_list *list;
 	struct community *com = NULL;
 	regex_t *regex = NULL;
+	int64_t seqnum = -1;
```

Comment:
A nit -- it might be better to #def something here to let future readers know what is going on, rather than assuming they will go figure out that "-1" means "nothing is assigned, so please assign something automatically..." Again, not a big thing, but it might prevent some small confusion in the future.